### PR TITLE
Handle builtin functions in isConst

### PR DESCRIPTION
**Pull request by sjoelund**

### CODE REVIEW

File: Compiler/FrontEnd/Expression.mo [hunk 1]
```diff
@@ -7966,6 +7967,10 @@ algorithm
         if res then isConstWork(e1) else false;
 
     case (DAE.CALL(expLst=ae, attr=DAE.CALL_ATTR(builtin=false, isImpure=false))) then isConstWorkList(ae);
+    case (DAE.CALL(path=path, expLst=ae, attr=DAE.CALL_ATTR(builtin=true))) then
+      if true or listMember(Absyn.pathFirstIdent(path),
```
Comment (hkiel) [inline Compiler/FrontEnd/Expression.mo]: why `if true` ???

Comment (sjoelund) [reply Compiler/FrontEnd/Expression.mo]: That's my old debugging code to see if it triggered :D

**Comment by OpenModelica-Hudson** [comment]
The test suite is unstable according to [OpenModelica_TEST_PULL_REQUEST 2018-04-05_11-48-39](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/5528/).

**Comment by OpenModelica-Hudson** [comment]
The tests run correctly according to [OpenModelica_TEST_PULL_REQUEST 2018-04-09_13-41-30](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/5540/console).

**Comment by sjoelund** [comment]
I'm not sure that's the best solution: ideally all constant subexpressions should be moved out and calculated only once (but naturally only calculated when needed)...

**Pull request closed without merging**